.gitignore

```diff
@@ -61,3 +61,14 @@ GTAGS
 
 # editor backup files etc.
 \#*\#
+
+# Python specific
+/install-deps-python2.7
+/install-deps-python3
+
+# Ceph-disk
+/src/ceph-disk/
+
+# Other
+/src/pybind/mgr/dashboard_v2/
```

Comment:
dashboard_v2 has been renamed. so we don't need this, do we?

Reply:
I was just basing this on my tree.  Any suggested improvements are welcome.

Reply:
I added those directories because they came up on "git status" as untracked.